app/models/cis2_info.rb

```diff
@@ -44,6 +46,10 @@ def can_supply_using_pgd?
     role_code == NURSE_ROLE
   end
 
+  def can_administer_without_prescription?
```

Comment:
Is this related to PSD? Would we use this instead of `can_prescribe_psd?`

Reply:
My understanding is we would use this once a PSD is in place to determine whether the user is allowed to perform the vaccination.

I wasn't entirely sure about the name, but I based it on the description of the activity code [in the ticket](https://nhsd-jira.digital.nhs.uk/browse/MAV-1365).

> Allows user to provide medication to a patient without issuing a prescription.